### ISSUE TITLE
Package not installable

### ISSUE THREAD

**Issue by timvink** [raeslab/lorepy#2]
```bash
pip install lorepy
```

Gives

```
Python 3.10.14 | packaged by conda-forge | (main, Mar 20 2024, 12:45:18) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import lorepy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'lorepy'
```

This is because the `setup.py` is not configured properly. When developing and running unit tests, you working directory is inside the project, so `import lorepy` is just loading the `lorepy` folder, not the package. I will provide a PR

**Comment by sepro** [comment]
Thank you for bringing this to my attention and providing a fix. Strange I didn't encounter this with pip/pypi myself. 

Will have a proper look Monday and push the new version to pypi. (And then apply this fix for two other repos where I used the same template 😂)

**Comment by sepro** [comment]
Wasn't able to replicate the issue (Python 3.11.5, fresh venv, pip install lorepy) but since having a `src` folder is considered best practice this was included in #5
